### DISCUSSION TITLE
Shorcuts to change focus window

### DISCUSSION THREAD

**Discussion by CirKanus** [domferr/tilingshell#228] [Ideas]
If you could implement some keyboard shortcuts to change the window in focus without using the mouse, this application would be perfect.

Just like this extension does.
https://extensions.gnome.org/extension/4627/focus-changer/

**Comment by domferr** [comment]
Hey, thank you sharing. Tiling Shell has already such feature, check out the key bindings in the extension preferences! Let me know if you don't find them

**Reply by CirKanus** [reply]
Found, thank you very much for your response; this application is perfect.
